### PR TITLE
add record and  dntr confirmation pages, using existing assets

### PR DESCRIPTION
**Pull request by xebu**

### CODE REVIEW

File: server/controllers/recommendation/apRecordDecisionController.test.ts [hunk 1]
```diff
@@ -9,19 +9,48 @@ jest.mock('../../monitoring/azureAppInsights')
...
       },
     })
     const next = mockNext()
     await apRecordDecisionController.get(mockReq(), res, next)
 
-    expect(res.locals.page).toEqual({ id: 'apRecordDecision' })
+    expect(res.locals.page).toEqual({
+      id: 'apRecordDecision',
+      bodyText:
+        'This will be recorded as a contact in NDelius called ‘Management oversight- recall’. You cannot undo this.',
```
Comment (DarrenW-HMPPS) [inline server/controllers/recommendation/apRecordDecisionController.test.ts:28]: Should be a space between oversight and hyphen.

File: server/views/pages/recommendations/apRationaleConfirmation.njk [hunk 2]
```diff
@@ -2,37 +2,39 @@
...
         <div class="govuk-grid-column-two-thirds-from-desktop">
             <div class="govuk-panel govuk-panel--confirmation">
-              <h1 class="govuk-panel__title">
-                Recall started
-              </h1>
-              <div class="govuk-panel__body" data-qa="fullName">
-              {{ personOnProbation }}
-              </div>
-              <div class="govuk-panel__body" data-qa="crn">
-                NOMIS number: {{ nomsNumber }}
```
Comment (DarrenW-HMPPS) [inline server/views/pages/recommendations/apRationaleConfirmation.njk:15]: Is getting rid of NOMIS number intentional?

**Review by DarrenW-HMPPS — CHANGES_REQUESTED** [review]

File: server/views/pages/recommendations/apRationaleConfirmation.njk (hunk 2 above)
Comment (xebu) [reply server/views/pages/recommendations/apRationaleConfirmation.njk:15]: Yes - I was informed on Friday that it is now the `crn` and not `nomis`

File: server/controllers/recommendation/apRecordDecisionController.test.ts (hunk 1 above)
Comment (xebu) [reply server/controllers/recommendation/apRecordDecisionController.test.ts:28]: Just realised the `management insight` text is incorrect. This has been changed to `‘Management oversight - no recall`

**Pull request merged**